qutebrowser/html/tabs.html

```diff
@@ -0,0 +1,46 @@
+{% extends "styled.html" %}
+
+{% block style %}
+{{super()}}
+h1 {
+    margin-bottom: 10px;
+}
+
+.url a {
+    color: #444;
+}
+
+th {
+    text-align: left;
+}
+
+.qmarks .name {
+    padding-left: 5px;
+}
+
+.empty-msg {
+    background-color: #f8f8f8;
+    color: #444;
+    display: inline-block;
+    text-align: center;
+    width: 100%;
+}
+{% endblock %}
+
+{% block content %}
+
+<h1>Tab list</h1>
+{% for win_id, tabs in tab_list_by_window.items() %}
+<h2>Window {{ win_id }}</h2>
+<table class="tabs_win{{win_id}}">
+    <tbody>
+    {% for name, url in tabs %}
+    <tr>
+        <td class="name"><a href="{{url}}">{{name}}</a></td>
+        <td class="url"><a href="{{url}}">{{url}}</a></td>
+    </tr>
+    {% endfor %}
+    </tbody>
+</table>
```

Comment:
You removed the raw list of tabs entirely - so you don't need that anymore for your script? Like I said, I'd be fine with having it on the page but hidden so an userscript can still read it.

Reply:
I re-added it, but will use the `QUTE_*` environment variables in the future.

Reply:
Using `<details>` is fine for me, but now it stick quite closely to the table which looks a bit weird. Maybe add some margin in the CSS there?